flink-runtime/src/test/java/org/apache/flink/runtime/highavailability/nonha/embedded/EmbeddedLeaderServiceTest.java

```diff
@@ -57,17 +49,18 @@ public void testConcurrentGrantLeadershipAndShutdown() throws Exception {
             leaderElection.startLeaderElection(contender);
             leaderElection.close();
```

Comment:
couldn't it happen that  `leaderElection.startLeaderElection` and `leaderElection.close` will not be called?

Reply:
If start() fails there's nothing to close.